tests/test_Executor.cc

```diff
@@ -293,7 +293,7 @@ TEST_CASE("Executor: Redirection of print() output", "[Executor]")
 TEST_CASE("Executor: Access context after run", "[Executor]")
 {
     Context ctx;
-    ctx.variables["a"] = VariableValue{ 1LL };
+    ctx.variables["a"] = StepVariable::Integer{ 1 };
```

Comment:
For me the test also set an example of the recommended idiom for `taskolib` users.

That is for example needed for `ctx.variables["b"] = StepVariable::String{ "C string literal" };` (on some old compilers).

I guess I do not really know what you want to say (or not, because there is the winky)